MapboxNavigation/Locale.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension Locale {
+    static var preferredLocalLanguageCountryCode: String {
+        let bundleLanguage = Bundle.main.preferredLocalizations.first!
+        let bundleLanguages = bundleLanguage.components(separatedBy: "-")
+        
+        if bundleLanguages.count == 2 {
+            return bundleLanguage
+        }
+        
+        let countryCode = NSLocale.Key.countryCode
+        let locale = NSLocale.current as NSLocale
```

Comment:
No need for a separate variable for the result of this cast. Surround it with parentheses and inline it below.